cdap-ui/app/cdap/components/PluginJSONCreator/Create/PluginJSONMenu/JSONActionButtons/LiveJSONButton/index.tsx

```diff
@@ -14,35 +14,55 @@
  * the License.
  */
 
-import Button from '@material-ui/core/Button';
+import * as React from 'react';
+
 import withStyles, { StyleRules } from '@material-ui/core/styles/withStyles';
-import Tooltip from '@material-ui/core/Tooltip';
+
 import CodeIcon from '@material-ui/icons/Code';
-import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import { LiveViewMode } from 'components/PluginJSONCreator/Create/PluginJSONMenu';
+import Tooltip from '@material-ui/core/Tooltip';
+import classnames from 'classnames';
 
 const styles = (theme): StyleRules => {
   return {
     buttonTooltip: {
-      fontSize: '14px',
-      backgroundColor: theme.palette.grey[500],
+      fontSize: '13px',
+      backgroundColor: theme.palette.grey[50],
+    },
+    liveJSONButton: {
+      '&:focus': {
+        outline: 'none',
+      },
+    },
+    clickedButton: {
+      backgroundColor: 'rgb(26,115,232,0.2)',
```

Comment:
Why customize the styles for the button and tooltip?

Reply:
<img width="57" alt="Screen Shot 2020-07-28 at 2 47 49 PM" src="https://user-images.githubusercontent.com/14116152/88708211-50339c00-d0e1-11ea-98e4-cedf1636886c.png">

I wanted to customize the styling to emphasize the currently selected button.

Reply:
I asked because I'm concerned about the one-off styling. Material UI has a `ToggleButton` in the lab https://material-ui.com/components/toggle-button/ but I don't know that we want to add the lab (yet). Is it worth adding a component for this?

Reply:
I've considered that alternative as well but decided against it since 1) we currently do not have @material-ui/lab and 2) even if we add ToggleButton we may need to add additional stylings.